viz.yaml

```diff
@@ -124,6 +123,25 @@ visualize:
     alttext: "Map of US state water use"
 publish:
   -
+    id: thumbnail
+    location: images/tmp_thumbnail_map.png
+    mimetype: image/png
+    title: thumbnail
+    alttext: "water use thumbnail"
+  -
+    id: usgs-logo
+    location: images/usgsLogo.png
+    mimetype: image/png
+    title: "USGS Logo"
+    alttext: "USGS Logo"
+  -
+    id: vizlab-favicon
+    location: images/favicon.ico
+    mimetype: image/png
+    comment: saying it is a png to get it in right spot, need handler for ico
```

Comment:
This is something we want to work into the vizlab package.  Maybe I'll try to work that in before release.